src/builder/QuarkOperationHelper.sol

```diff
@@ -101,6 +101,7 @@ library QuarkOperationHelper {
         IQuarkWallet.QuarkOperation memory lastQuarkOperation = quarkOperations[quarkOperations.length - 1];
         IQuarkWallet.QuarkOperation memory mergedQuarkOperation = IQuarkWallet.QuarkOperation({
             nonce: lastQuarkOperation.nonce,
+            isReplayable: lastQuarkOperation.isReplayable,
```

Comment:
Same here, so basically if it's `wrap` and `replayableSupply` this is now `replayableWrapAndSupply`?

Reply:
Yeah, if the last operation is replayable, the merged operation should be replayable.